docs/products/kafka/kafka-connect/howto/gcp-bigquery-sink.rst

```diff
@@ -0,0 +1,245 @@
+Create a Google BigQuery sink connector
+=======================================
+
+The Google BigQuery sink connector enables you to move data from an Aiven for Apache Kafka® cluster to a set of Google BigQuery tables for further processing and analysis. 
```

Comment:
Link to source?

Reply:
and their docs/config also

Reply:
Added link to source and, after the parameters discussion the link to all the parameters.